Habitica/src/main/java/com/habitrpg/android/habitica/ui/activities/ClassSelectionActivity.kt

```diff
@@ -292,4 +300,11 @@ class ClassSelectionActivity : BaseActivity() {
             finish()
         }
     }
+
+    private fun checkForReviewPromptAfterClassSelection() {
+        userViewModel.user.observeOnce(this) { user ->
+            val totalCheckIns = user?.loginIncentives ?: return@observeOnce

```

Comment:
This version also works (see my other comment. But I think we should stick to one way.